test/integration/models/database/test_database.py

```diff
@@ -165,6 +165,40 @@ def test_database_instance(test_linode_client, test_create_sql_db):
     assert str(test_create_sql_db.id) in str(dbs.lists)
 
 
+@pytest.mark.skipif(
+    os.getenv("RUN_DB_TESTS", "").strip().lower() not in {"yes", "true"},
+    reason="RUN_DB_TESTS environment variable must be set to 'yes' or 'true' (case insensitive)",
+)
+def test_mysql_suspend_resume(test_linode_client, test_create_sql_db):
+    db = test_linode_client.load(MySQLDatabase, test_create_sql_db.id)
+
+    db.suspend()
+
+    wait_for_condition(
+        10,
+        300,
+        get_sql_db_status,
+        test_linode_client,
+        test_create_sql_db.id,
+        "suspended",
+    )
```

Comment:
Just a heads up for the future, we'll probably want to replace this with event polling once the update event issue is resolved in the API. I don't have an ETA for that so this should be fine for now 👍